src/pages/resources/security.mdx

```diff
@@ -1,4 +1,4 @@
-# Bug bounty
```

Comment:
Do we want to leave the term bug bounty anywhere? How do other teams approach this and label it? I think some folks will be looking for bounty.

Reply:
Yeah, reporters search for this. We should preserve that (maybe add it in brackets, or add another reference using it that links to this page). Can you also verify if this page is easily reachable via google search?

Reply:
I did a search for the following terms on both my work and personal laptops:

- "axelar bug bounty"
- "axelar report security bug"
- "axelar report security risk"
- "axelar report security"
- "axelar report security vulnerability"

This page didn't pop up on any of those searches. Most of them led directly to our [Immunefi page](https://immunefi.com/bounty/axelarnetwork/), or to the ["Security on Axelar Network"](https://axelar.network/blog/security-at-axelar-core) blog post.

I'm ok with putting "Bug Bounty" in brackets. Will do this now.